src/services/MQTTRoomService/MQTTRoomService.ts

```
@@ -0,0 +1,115 @@
+    const {
+      [PostMessageStructKey.MessageId]: messageId,
+      [PostMessageStructKey.Success]: success,
+      [PostMessageStructKey.Result]: result,
+      [PostMessageStructKey.Error]: error,
+      [PostMessageStructKey.EnvelopeType]: envelopeType,
+      [PostMessageStructKey.EventName]: eventName,
+      [PostMessageStructKey.PeerId]: peerId,
+    } = event.data;
+
+    if (envelopeType === EnvelopeType.Function) {
+      if (messageId in this.messagePromises) {
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)